### ISSUE TITLE
Audio Sync Issues after rendering

### ISSUE THREAD

**Issue by jreuter** [jliljebl/flowblade#553]
Sorry for the duplicate.  I forgot to log out of my work github before posting.

Please include the following information.

    Flowblade version (Help->About):
    1.16.0
    MLT version (Help->Runtime Environment):
    6.0.0
    Your distribution (Ubuntu, Debian, Mint etc.):
    Mint 18.1 and Mint 18.3

File for testing: https://www.dropbox.com/s/ts3x3ozxjetf304/The_Brink-s01e01-cut.mp4?dl=0

To duplicate:
Open new project, add the above file and allow project update to HD 1080p, 29.97fps. Drop in timeline and render as MPEG-4 at 10000 kb/s. Notice audio/video is fine at the beginning of the file, but way off at the end.

There is a hiccup in the original video just before 6:09 that may be the root cause.

Let me know if you need anything else to help track down the issue. I have a bunch of TV recordings that cause this issue.

**Comment by dvdlvr** [comment]
This is likely due to melt not being able to keep the sync after a problem with the source file.
I've created a script that will do a workaround. It will increase the render time, but will keep the sync after the problem area in the source file.
See my post here:
https://plus.google.com/u/0/106068904231748287798/posts/7QdGJnzWiPT

I don't see how this could be implemented in Flowblade without huge changes to the way it works, which would also affect the render time tremendously.
As I see it, the problem is inherent to melt.

Let me know if the script works for you.

**Comment by jliljebl** [comment]
This is an MLT issue and can't be on Flowblade side. There was  a bug report on MLT list which seemed likely to about this issue. I'll leave this open for info since there has been a workaround offered here.

**Comment by user23498723452** [comment]
After reviewing the MLT bug list , a similar sounding bug appears to have been fixed shortly after the above script was created.  Since I can't even fathom how to use the script is there any chance that the updated version of mlt can be included in a new version of flowblade?

**Comment by dvdlvr** [comment]
To use the script, follow these steps:
1) Create your edit in Flowblade
2) File -> Export -> MLT XML -> Save with name /saved_xml.xml/ in the 
directory where the script is installed.
3) Open up a command prompt, go to the directory where the script is 
installed and run:
./Multicut.sh   1080 /saved_xml.xml/   100   0000-Chunk FinalOutput

This will create a file called FinalOutput.mp4, based on the 
saved_xml.xml file with default parameters for 1080 content.
Hope this helps.

**Comment by neutrinog** [comment]
Would love to see this fixed in some fashion some day.

**Comment by jliljebl** [comment]
This is not reproducing here with file provided by @jreuter 

I'm marking this unreproduced but I'll leave open to see if we can get further reproductions.

@neutrinog which version MLT you are using? MLT needs to be 6.16 all possible bug fixes related to this to available.

**Comment by neutrinog** [comment]
@jliljebl 6.16. But the last time I used flowblade was oct 9th, so if that's a recent update I haven't had a chance to use it yet.

**Comment by jliljebl** [comment]
This is not moving forward and in my test there was a glitch in the source file, closing as *unreproduced* i do not believe it is propable that we can find improvement investigating this particular Issue.

**Comment by tjcaul** [comment]
I had a similar issue and solved it by exporting to MLT XML format in Flowblade, then using the `melt` command line tool to render to FFmpeg (`avformat` option`), which saves to a file. Output file options can be configured according to the [MLT documentation](https://www.mltframework.org/plugins/ConsumerAvformat/).
```
$ melt --version
melt 7.30.0
$ melt -progress file.xml -consumer avformat:output.mp4
```
I didn't need to use the script provided by @dvdlvr; presumably the sync issue is fixed in Melt 7.30.0. I'm not sure why this works, because Flowblade using Melt 7.30.0 still exhibits the bug.
